packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -331,4 +420,56 @@ describe("ActionMenu", () => {
             expect(wrapper.find(ActionMenu).prop("opened")).toBe(false);
         });
     });
+
+    describe("Custom Opener", () => {
+        it("opens the menu when clicking on the custom opener", () => {
+            // Arrange
+            const menu = mount(
+                <ActionMenu
+                    menuText={"Action menu!"}
+                    testId="openTest"
+                    onChange={onChange}
+                    selectedValues={[]}
+                    opener={(eventState) => (
+                        <button aria-label="Search" onClick={jest.fn()} />
+                    )}
```

Comment:
We probably don't need to test the values of `eventState` because they're being passed directly from `ClickableBehavior`, right?

Reply:
Right, and this should be already tested in `ClickableBehavior`.